<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveFont</class>
 <widget class="QMainWindow" name="CurveFont">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CurveFont</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>102</y>
      <width>120</width>
      <height>116</height>
     </rect>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>历史文件</string>
     </property>
     <addaction name="actionHistoryFile0"/>
     <addaction name="actionHistoryFile1"/>
     <addaction name="actionHistoryFile2"/>
     <addaction name="actionHistoryFile3"/>
     <addaction name="actionHistoryFile4"/>
     <addaction name="actionHistoryFile5"/>
     <addaction name="actionHistoryFile6"/>
     <addaction name="actionHistoryFile7"/>
     <addaction name="actionHistoryFile8"/>
     <addaction name="actionHistoryFile9"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menu_2"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionTchinese"/>
   </widget>
   <addaction name="File"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>简体中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>英文</string>
   </property>
  </action>
  <action name="actionTchinese">
   <property name="text">
    <string>繁体中文</string>
   </property>
  </action>
  <action name="actionHistoryFile0">
   <property name="text">
    <string>HistoryFile0</string>
   </property>
  </action>
  <action name="actionHistoryFile1">
   <property name="text">
    <string>HistoryFile1</string>
   </property>
  </action>
  <action name="actionHistoryFile2">
   <property name="text">
    <string>HistoryFile2</string>
   </property>
  </action>
  <action name="actionHistoryFile3">
   <property name="text">
    <string>HistoryFile3</string>
   </property>
  </action>
  <action name="actionHistoryFile4">
   <property name="text">
    <string>HistoryFile4</string>
   </property>
  </action>
  <action name="actionHistoryFile5">
   <property name="text">
    <string>HistoryFile5</string>
   </property>
  </action>
  <action name="actionHistoryFile6">
   <property name="text">
    <string>HistoryFile6</string>
   </property>
  </action>
  <action name="actionHistoryFile7">
   <property name="text">
    <string>HistoryFile7</string>
   </property>
  </action>
  <action name="actionHistoryFile8">
   <property name="text">
    <string>HistoryFile8</string>
   </property>
  </action>
  <action name="actionHistoryFile9">
   <property name="text">
    <string>HistoryFile9</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
